base/utferror.jl

```diff
@@ -0,0 +1,30 @@
+# This file is a part of Julia. License is MIT: http://julialang.org/license
+
+##\brief      Error messages for Unicode / UTF support
```

Comment:
https://github.com/JuliaLang/julia/pull/11551#discussion_r31588648
https://github.com/JuliaLang/julia/pull/11573#discussion_r31721662

Has anyone ever said "yes using doxygen style comments in Julia code is awesome," or just expressed approval to having more comments in general? The latter I can believe, the former would surprise me.